packages/teams-js/src/internal/communication.ts

```diff
@@ -267,21 +267,28 @@ function sendMessageToParentHelper(actionName: string, args: any[]): MessageRequ
   return request;
 }
 
+const processMessageLogger = communicationLogger.extend('processMessage');
+
 /**
  * @internal
  * Limited to Microsoft-internal use
  */
 function processMessage(evt: DOMMessageEvent): void {
   // Process only if we received a valid message
   if (!evt || !evt.data || typeof evt.data !== 'object') {
+    processMessageLogger('Unrecognized message format received by app, message being ignored');
```

Comment:
>'Unrecognized message format received by app, message being ignored' [](http://example.com/codeflow?start=25&length=68)

Could we log the evt object here to help aid in debugging?

Reply:
Done